ocs_ci/utility/aws.py

```diff
@@ -309,3 +309,33 @@ def start_ec2_instance(self, instance_id, wait=False):
             return False
         state = res.get('StartingInstances')[0].get('CurrentState').get('Code')
         return state == PENDING
+
+    def get_volumes_attached_to_instance(self, instance_id):
+        """
+        Get a list of volumes attached to a given instance
+
+        Args:
+            instance_id (str): ID of the instance
+
+        Returns:
+            volumes: List, of volumes attached to the instance
+        """
+        logger.info(instance_id)
+        volumes_response = self.ec2_client.describe_volumes(
+            Filters=[
+                {
+                    'Name': 'attachment.instance-id',
+                    'Values': [instance_id],
+                },
+            ],
+        )
+        volumes = []
+        for volume in volumes_response['Volumes']:
+            volumes.append(
+                dict(
+                    id=volume['VolumeId'],
+                    attachments=volume['Attachments']
+                )
+            )
+        logger.info(volumes)
+        return volumes
```

Comment:
add a new empty line - pep8 thing to have empty line at the end of every .py file